libcaf_core/caf/flow/step/ignore_elements.hpp

```diff
@@ -0,0 +1,46 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#pragma once
+
+#include "caf/detail/ring_buffer.hpp"
+#include "caf/fwd.hpp"
+
+#include <cstddef>
+
+namespace caf::flow::step {
+
+template <class T>
+class ignore_elements {
+public:
+  using input_type = T;
+
+  using output_type = T;
+
+  explicit ignore_elements() {
+    // nop
+  }
+
+  ignore_elements(ignore_elements&&) = default;
+  ignore_elements(const ignore_elements&) = default;
+  ignore_elements& operator=(ignore_elements&&) = default;
+  ignore_elements& operator=(const ignore_elements&) = default;
+
+  template <class Next, class... Steps>
+  bool on_next(const input_type& item, Next& next, Steps&... steps) {
```

Comment:
Don't you get warnings for unused parameters on MSVC here? If not, we need to figure out the appropriate flags.

```suggestion
  bool on_next(const input_type&, Next&, Steps&...) {
```

Reply:
I don't see any warning for unused parameters. I will look into appropriate flags for MSVC for this.